packages/mobile/src/harmony-native/components/layout/Box/Box.tsx

```diff
@@ -94,10 +94,10 @@ export const Box = styled(View, {
         // Using white as a default here, but can be overridden
         backgroundColor: color.background.white
       }),
-      paddingTop: padT && spacing[padT],
-      paddingLeft: padL && spacing[padL],
-      paddingRight: padR && spacing[padR],
-      paddingBottom: padB && spacing[padB],
+      paddingTop: padT && (spacing[padT] ?? padT),
+      paddingLeft: padL && (spacing[padL] ?? padL),
+      paddingRight: padR && (spacing[padR] ?? padR),
+      paddingBottom: padB && (spacing[padB] ?? padB),
```

Comment:
is this to allow passing in discrete values? I feel like this was kind of inevitable, but might conflict with the type being the strings for the sizes.

Reply:
Also we should make margin and other size based props consistent with this

Reply:
yeah... i do think it is nice, in the case for this pr, we added the "safe-area-view's" paddingbottom, lmk if you think we should just do style object for those cases though?

yeah ill go through the rest of the props after this and make everything consistent.

Reply:
yea i feel like we should try to stick to the agreed spacing sizes that design set out so doing this feels like we're just allowing whatever to go in here. That said, it's a bit idealistic and we do have to do a lot of css objects to create padding that are not on our grid so i figured this would be added eventually 🤷